packages/artalk/src/components/list-lite.ts

```diff
@@ -472,14 +473,14 @@ export default class ListLite extends Component {
   }
 
   /** 版本检测 */
-  public versionCheck(name: '前端'|'后端', needVersion: string, curtVersion: string): boolean {
+  public versionCheck(name: 'frontend'|'backend', needVersion: string, curtVersion: string): boolean {
     const needUpdate = Utils.versionCompare(needVersion, curtVersion) === 1
     if (needUpdate) {
       // 需要更新
-      const errEl = Utils.createElement(`<div>Artalk ${name}版本已过时，请更新以获得完整体验<br/>`
+      const errEl = Utils.createElement(`<div>Artalk ${this.$t(name)}版本已过时，请更新以获得完整体验<br/>`
```

Comment:
Is it forgotten here or are there design considerations?

Reply:
the i18n of artalk backend is still WIP, i gonna change it after backend is done 🤔🤔